src/dawn/engine_dawn.cc

```diff
@@ -359,13 +356,30 @@ Result EngineDawn::DoClear(const ClearCommand* command) {
   color_attachment.clearColor = render_pipeline->clear_color_value;
   color_attachment.loadOp = ::dawn::LoadOp::Clear;
   color_attachment.storeOp = ::dawn::StoreOp::Store;
-
-  // Attach that colour attachment to the render pass.
   ::dawn::RenderPassColorAttachmentDescriptor* rpca[] = {&color_attachment};
```

Comment:
Can we rename rpca to color_attachment_descriptor to make it clearer what this is?

Reply:
Yes that's better!